crates/oxc_linter/src/rules/vitest/prefer_each.rs

```diff
@@ -0,0 +1,257 @@
+use oxc_ast::AstKind;
+use oxc_diagnostics::OxcDiagnostic;
+use oxc_macros::declare_oxc_lint;
+use oxc_semantic::{AstNode, AstNodeId};
+use oxc_span::{GetSpan, Span};
+
+use crate::{
+    context::LintContext,
+    rule::Rule,
+    utils::{parse_jest_fn_call, JestFnKind, JestGeneralFnKind, PossibleJestNode},
+};
+
+fn use_prefer_each(span0: Span, fn_name: &str) -> OxcDiagnostic {
```

Comment:
```suggestion
fn use_prefer_each(span: Span, fn_name: &str) -> OxcDiagnostic {
```

Reply:
Do we need to refactor the previous related code?

Reply:
If necessary, I can complete the refactoring of all `span0` in one PR.

Reply:
Yes. Those were added due to a code mod. the number suffixes were to avoid name collisions.